cmd/kubeadm/app/master/manifests.go

```diff
@@ -244,7 +243,7 @@ func getComponentCommand(component string, s *kubeadmapi.KubeadmConfig) (command
 			"--insecure-experimental-approve-all-kubelet-csrs-for-group=system:kubelet-bootstrap",
 		},
 		scheduler: {
-			// TODO(phase1+): consider adding --address=127.0.0.1 in order to not expose the scheduler port to the rest of the world
+			"--address=127.0.0.1",
```

Comment:
quoting Mike's comment here for context

> controller-manager and scheduler rest handlers expose basically nothing.

sounds like we can discuss adding this change later? I don't feel confident reviewing this change as-is, because I don't deeply understand the implications (and currently it's hard for me to test `kubeadm` changes, because it depends on a build env @errordeveloper has set up).